src/js/extension-anchor.js

```diff
@@ -24,6 +24,10 @@ var AnchorExtension;
 
         // Button and Extension handling
 
+        // labels for the anchor-edit form buttons
+        formSaveLabel: '&#10003;',
+        formCloseLabel: '&times;',
```

Comment:
it speaks to the oddity that is `options.buttonLabels` having that special "fontawesome" path everywhere. You either want to override every individual label, or use the entirety of the pre-defined values for 'fontawesome' icon set.  not sure the best balance there?